modules/ROOT/pages/until-successful-scope.adoc

```diff
@@ -4,7 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, studio, mule, until successful, reattempts, retry
 
-The Until Successful scope processes the components within it until they succeed or exhaust the maximum number of retries.
+The Until Successful scope processes the components within it until they
+succeed or exhaust the maximum number of retries. Like all Core components
+other than Async, Until Successful runs synchronously. However, it blocks
```

Comment:
i would avoid using the work `block` here since it may be interpreted as threads are blocked which is not the case, and contradict the premises of a non-blocking engine.